api/v1beta1/bgpadvertisement_webhook.go

```diff
@@ -56,8 +56,13 @@ func (bgpAdv *BGPAdvertisement) ValidateCreate() error {
 		return err
 	}
 
+	communities, err := getExistingCommunities()
+	if err != nil {
+		return err
+	}
+
 	toValidate := bgpAdvListWithUpdate(existingBGPAdvList, bgpAdv)
-	err = Validator.Validate(toValidate, addressPools, ipPools)
+	err = Validator.Validate(toValidate, addressPools, ipPools, communities)
```

Comment:
Since we have said we should not check for the presence of the community alias, I wonder if we should remove this whole commit

Reply:
I thought about it and I think we should.
The bgpAdv webhook should be able to handle the communities part
of the spec without the Community CRs.

Reply:
Probably the same for addressPool

Reply:
Nope, addresspool is different because it makes sense to validate the consistency with what we know it's there. It's not about validating that something that should be there is not.